### DISCUSSION TITLE
Restock Gems [PROPOSAL]

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#4467] [Ideas]
Due to the introduction of gems into the game, I propose to change the restock system. 
Inspired by Dcol idea:https://github.com/sunflower-land/sunflower-land/discussions/3773 
We need to divide seeds and tools into categories that will be replenished independently of each other.

Categories:
### Seeds
- Basic crops = 2 Gems 
- Medium crops = 3 Gems 
- Advanced crops = 4 Gems 
- Fruit = 3 Gems 
- Flowers = 3 Gems 
- Greenhouse = 5 Gems 
### Tools
- Basic tools  (Axe,Pickaxe, Stone Pickaxe, Iron Pickaxe) = 5 Gems 
- Rod = 3 Gems 
- Advanced tools(Gold Pickaxe, Oil Drill) = 7 Gems 
- Jafar Tools = 5 Gems 
_Values are approximate._

At the expense of Gems you can make small prices for each category, this will allow both new players to get into the game faster and experienced players to replenish more often.

**Comment by Alucard0389** [comment]
I like the idea because I running mush faster out of tools. And can't use every seed I bought. Hopefully we get something like this
